docs/design.html

```diff
@@ -366,7 +366,7 @@ <h4><a id="design_replicatedlog" href="#design_replicatedlog">Replicated Logs: Q
     <p>
     There are a rich variety of algorithms in this family including ZooKeeper's
     <a href="http://web.archive.org/web/20140602093727/http://www.stanford.edu/class/cs347/reading/zab.pdf">Zab</a>,
-    <a href="https://ramcloud.stanford.edu/wiki/download/attachments/11370504/raft.pdf">Raft</a>,
+    <a href="https://www.usenix.org/conference/atc14/technical-sessions/presentation/ongaro">Raft</a>,
```

Comment:
Thanks for the spotting the fix. 
Minor suggestion, I think it'd be best to link directly to the paper (pdf) instead of the page of the presentation (that indeed includes the paper too). 

Here are two options: 
https://web.stanford.edu/~ouster/cgi-bin/papers/raft-atc14
or
https://www.usenix.org/system/files/conference/atc14/atc14-paper-ongaro.pdf

Reply:
hi @kkonstantine , 
Thanks for the comments. I've updated the link to the pdf link: https://www.usenix.org/system/files/conference/atc14/atc14-paper-ongaro.pdf

Thanks.